planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -586,7 +586,29 @@ void BehaviorPathPlannerNode::run()
   const auto output = bt_manager_->run(planner_data);
 
   // path handling
-  const auto path = getPath(output, planner_data);
+  auto path = getPath(output, planner_data);
+
+  // check if path orientation is valid
+  if (!path->points.empty()) {
+    const auto closest_path_point_idx =
+      motion_utils::findNearestIndex(path->points, planner_data->self_pose->pose.position);
```

Comment:
It seems better to consider an overlapping path?
https://github.com/autowarefoundation/autoware.universe/blob/main/common/motion_utils/README.md#nearest-index-for-the-ego

Reply:
Right. Please use this function.
https://github.com/autowarefoundation/autoware.universe/blob/778f8c087ef7e5421891a92bd9ece563f06b232a/planning/behavior_path_planner/include/behavior_path_planner/behavior_path_planner_node.hpp#L183